perfkitbenchmarker/vm_util.py

```diff
@@ -95,6 +98,11 @@ class IpAddressSubset(object):
                   'the receiving VM is reachable by internal IP (REACHABLE), '
                   'external IP only (EXTERNAL) or internal IP only (INTERNAL)')
 
+flags.DEFINE_enum('background_network_iptype', IpAddressSubset.EXTERNAL,
```

Comment:
```--ip_addresses``` strikes me as a confusing flag name, since it suggests the actual address values. I vote ```ip_type``` but don't feel strongly about this issue.